.changeset/clever-eagles-unite.md

```diff
@@ -0,0 +1,5 @@
+---
+'openzeppelin-solidity': major
+---
+
+`PaymentSplitter`: removed.
```

Comment:
This is not a great changeset. We had a similar unresolved conversation with the previous round of deletions.

Maybe we should first think about what we want the changelog to look like, and from there we should figure out what changesets we need...

What I imagine is a single section called "Removals" with a section for each of the removal reasons. If so, I think we can have one changeset for each of those sections.

Reply:
Yes, I'll start that section in this PR, and we can move changesets from previous PR to that section in a dedicated PR.